okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.kt

```diff
@@ -38,6 +39,7 @@ import okhttp3.internal.tls.CertificateChainCleaner
 import okhttp3.internal.tls.TrustRootIndex
 
 /** Android 5+. */
+@SuppressSignatureCheck
```

Comment:
This is the highest risk part of the change, means watching Intellij/Android Studio warnings and running tests before releases generally.  We should automate the latter at some point.

Reply:
I wonder what the work would be to get “warning clean”. Might just end up dumping `@SuppressWarnings` everywhere, which I don’t love.